styles/components/_buttons.scss

```diff
@@ -325,6 +325,30 @@
   }
 }
 
+.btnFlx {
```

Comment:
This new button type seems different enough from our existing row buttons to warrant a separate name/style, but let me know if you think otherwise.